src/send_push/logger.ts

```diff
@@ -0,0 +1,19 @@
+import winston from 'winston';
```

Comment:
What is this file for? Can we [re-use the existing logger](https://github.com/customerio/cio-sdk-tools/blob/main/src/doctor/utils/logger.ts)?

Reply:
didnt want to increase the scope by trying to pull the logger into a common utility with this pr.

Reply:
Thanks for considering scope on a PR. 

I do want to avoid having multiple loggers in a project. Re-using the existing logger makes the most sense to me at this time. If that is done in this PR or another, I don't have a preference.